modules/installation-creating-aws-dns.adoc

```diff
@@ -6,12 +6,11 @@
 [id="installation-creating-aws-dns_{context}"]
 = Creating networking and load balancing components in AWS
 
-You must configure networking and load balancing (classic or network) in Amazon Web Services (AWS) for your
-{product-title} cluster to use. The easiest way to create these components is
-to modify the provided CloudFormation template, which also creates a hosted zone
-and subnet tags.
+You must configure networking and load balancing (classic or network) in Amazon Web Services (AWS) for your {product-title} cluster to use.
```

Comment:
ISG: avoid parentheses
Also, the end of the sentence ("to use") sounds like it's missing something, although I'm sure it's not. Perhaps try rewording to something like: "You must configure networking and classic or network load balancing in Amazon Web Services (AWS) that your {product-title} cluster can use."

Reply:
Thank you for the suggestion, this sounds good to me. Resolved.